### PR TITLE
Improve timesyncd example

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Review by mauromorales — APPROVED** [review]

File: content/en/docs/Reference/stage_modules.md [hunk 1]
```diff
@@ -273,15 +273,16 @@ Sets the `systemd-timesyncd` daemon file (`/etc/system/timesyncd.conf`) file acc
...
-         - systemd-timesyncd
-       timesyncd:
+    boot:
+      - name: "Setup NTP"
+        systemctl:
+          enable:
+            - systemd-timesyncd
+        timesyncd:
           NTP: "0.pool.org foo.pool.org"
-          FallbackNTP: ""
```
Comment (kreeuwijk) [inline content/en/docs/Reference/stage_modules.md:283]: please leave this in but with a value:
```
FallbackNTP: "us.pool.ntp.org"
```

**Comment by kreeuwijk** [comment]
correct spacing would be:
```
stages:
    boot:
      - name: "Setup NTP"
        systemctl:
            enable:
              - systemd-timesyncd
        timesyncd:
            NTP: "0.pool.org foo.pool.org"
            FallbackNTP: "us.pool.ntp.org"
      - name: "Restart NTP service so it gets the new config"
        commands:
          - systemctl restart systemd-timesyncd
```

**Comment by Itxaka** [comment]
> correct spacing would be:
> 
> ```
> stages:
>     boot:
>       - name: "Setup NTP"
>         systemctl:
>             enable:
>               - systemd-timesyncd
>         timesyncd:
>             NTP: "0.pool.org foo.pool.org"
>             FallbackNTP: "us.pool.ntp.org"
>       - name: "Restart NTP service so it gets the new config"
>         commands:
>           - systemctl restart systemd-timesyncd
> ```

Is see 4 spaces on my 90_custom file, yours is mixing 4 and 2. In any case both are valid?

```
install:
    device: /dev/vda
k3s:
    enabled: false
name: Config generated by the installer
stages:
    itxaka:
        - timesyncd:
            NTP: "caca.pool.org"
            FallbackNTP: ""
    initramfs:
        - users:
            kairos:
                groups:
                    - admin
                name: kairos
                passwd: kairos
```

**Comment by kreeuwijk** [comment]
Ah I see you changed everything to 2 spaces. That's fair too

**Review by kreeuwijk — APPROVED** [review]
LGTM

**Pull request merged**